hack/kind-cluster-v1.12.8.sh

```diff
@@ -0,0 +1,172 @@
+  - containerPath: /mnt/disks/vol2
+    hostPath: ${data_dir}/worker2/vol2
+  - containerPath: /mnt/disks/vol3
+    hostPath: ${data_dir}/worker2/vol3
+  - containerPath: /mnt/disks/vol4
+    hostPath: ${data_dir}/worker2/vol4
+  - containerPath: /mnt/disks/vol5
+    hostPath: ${data_dir}/worker2/vol5
+  - containerPath: /mnt/disks/vol6
+    hostPath: ${data_dir}/worker2/vol6
+  - containerPath: /mnt/disks/vol7
+    hostPath: ${data_dir}/worker2/vol7
+  - containerPath: /mnt/disks/vol8
+    hostPath: ${data_dir}/worker2/vol8
+  - containerPath: /mnt/disks/vol9
+    hostPath: ${data_dir}/worker2/vol9
+- role: worker
+  extraMounts:
+  - containerPath: /mnt/disks/vol1
+    hostPath: ${data_dir}/worker3/vol1
+  - containerPath: /mnt/disks/vol2
+    hostPath: ${data_dir}/worker3/vol2
+  - containerPath: /mnt/disks/vol3
+    hostPath: ${data_dir}/worker3/vol3
+  - containerPath: /mnt/disks/vol4
+    hostPath: ${data_dir}/worker3/vol4
+  - containerPath: /mnt/disks/vol5
+    hostPath: ${data_dir}/worker3/vol5
+  - containerPath: /mnt/disks/vol6
+    hostPath: ${data_dir}/worker3/vol6
+  - containerPath: /mnt/disks/vol7
+    hostPath: ${data_dir}/worker3/vol7
+  - containerPath: /mnt/disks/vol8
+    hostPath: ${data_dir}/worker3/vol8
+  - containerPath: /mnt/disks/vol9
+    hostPath: ${data_dir}/worker3/vol9
+- role: worker
+  extraMounts:
+  - containerPath: /mnt/disks/vol1
+    hostPath: ${data_dir}/worker4/vol1
+  - containerPath: /mnt/disks/vol2
+    hostPath: ${data_dir}/worker4/vol2
+  - containerPath: /mnt/disks/vol3
+    hostPath: ${data_dir}/worker4/vol3
+  - containerPath: /mnt/disks/vol4
+    hostPath: ${data_dir}/worker4/vol4
+  - containerPath: /mnt/disks/vol5
+    hostPath: ${data_dir}/worker4/vol5
+  - containerPath: /mnt/disks/vol6
+    hostPath: ${data_dir}/worker4/vol6
+  - containerPath: /mnt/disks/vol7
+    hostPath: ${data_dir}/worker4/vol7
+  - containerPath: /mnt/disks/vol8
+    hostPath: ${data_dir}/worker4/vol8
+  - containerPath: /mnt/disks/vol9
+    hostPath: ${data_dir}/worker4/vol9
+- role: worker
+  extraMounts:
+  - containerPath: /mnt/disks/vol1
+    hostPath: ${data_dir}/worker5/vol1
+  - containerPath: /mnt/disks/vol2
+    hostPath: ${data_dir}/worker5/vol2
+  - containerPath: /mnt/disks/vol3
+    hostPath: ${data_dir}/worker5/vol3
+  - containerPath: /mnt/disks/vol4
+    hostPath: ${data_dir}/worker5/vol4
+  - containerPath: /mnt/disks/vol5
+    hostPath: ${data_dir}/worker5/vol5
+  - containerPath: /mnt/disks/vol6
+    hostPath: ${data_dir}/worker5/vol6
+  - containerPath: /mnt/disks/vol7
+    hostPath: ${data_dir}/worker5/vol7
+  - containerPath: /mnt/disks/vol8
+    hostPath: ${data_dir}/worker5/vol8
+  - containerPath: /mnt/disks/vol9
+    hostPath: ${data_dir}/worker5/vol9
+EOF
+
+echo "start to create k8s cluster"
+kind create cluster --config kind-config.yaml --image kindest/node:${k8s_version} --name=${clusterName}
+export KUBECONFIG="$(kind get kubeconfig-path --name=${clusterName})"
```

Comment:
`KUBECONFIG` will not be available when script exited. you can print some instructions in the log, see [example](https://github.com/kubernetes/kubernetes/blob/8dea3310e532fcb28eed84c2bb9a16a6cb7cbfdf/hack/local-up-cluster.sh#L920-L936).

Reply:
expose `KUBECONFIG ` just for the following commands can execute normally.
and it is nice to print instructions in the stdout